pkg/reconciler/openshift/tektonaddon/extension.go

```diff
@@ -87,31 +87,28 @@ func (oe openshiftExtension) PostReconcile(ctx context.Context, comp v1alpha1.Te
 	logger := logging.FromContext(ctx)
 	addon := comp.(*v1alpha1.TektonAddon)
 
-	exist, err := checkIfInstallerSetExist(ctx, oe.operatorClientSet, oe.version, addon, miscellaneousResourcesInstallerSet)
+	exist, err := checkIfInstallerSetExist(ctx, oe.operatorClientSet, oe.version,
+		fmt.Sprintf("%s=%s", tektoninstallerset.InstallerSetType, MiscellaneousResourcesInstallerSet))
 	if err != nil {
 		return err
 	}
 	if !exist {
-
 		manifest, err := getMiscellaneousManifest(ctx, addon, oe.manifest, comp)
 		if err != nil {
 			return err
 		}
 
 		if err := createInstallerSet(ctx, oe.operatorClientSet, addon, manifest, oe.version,
-			miscellaneousResourcesInstallerSet, "addon-openshift"); err != nil {
-			return err
+			MiscellaneousResourcesInstallerSet, "addon-openshift"); err != nil {
+			return v1alpha1.RECONCILE_AGAIN_ERR
```

Comment:
if err is nil then return `v1alpha1.RECONCILE_AGAIN_ERR` 
else return err.. 

otherwise we will never catch the error

Reply:
:woman_facepalming:  :see_no_evil: